pyScienceMode2/enums.py

```diff
@@ -47,3 +47,77 @@ class Type(Enum):
     MotomedCommandDone = 89
     GetMotomedModeAck = 13
     GetMotomedMode = 12
+
+
+class Types(Enum):
```

Comment:
Change name

Reply:
Done.

Reply:
.

Reply:
TypeRehaP24 and TypeReha2 : better name ?